openedx/core/djangoapps/plugin_api/views.py

```diff
@@ -86,14 +117,18 @@ def render_standalone_response(self, request, fragment, **kwargs):
         """
         if fragment is None:
             return HttpResponse(status=204)
-        context = {
-            'uses-pattern-library': self.USES_PATTERN_LIBRARY,
+        context = self.create_base_standalone_context(request, fragment, **kwargs)
+        self._add_studio_context_variables(request, context)
```

Comment:
Yes, that's correct. I probably should put a clearer comment on the helper method that this is a short term hack. I don't know why Studio templates expect these context variables to be in place.